### PR TITLE
[PAGE] Create Bids page

### PR DESCRIPTION
**Pull request by alfakini**
In this PR, we introduce the Bids page. It is the starting point for the pages related to the human journey.
In the following PRs, the components will be generalized.

Screen Shot 2021-09-27 at 21 19 45

### CODE REVIEW

**Comment by alfakini** [comment]
[[PAGE] Create page Bids/List as human](https://trello.com/c/MMGLvBOq/207-page-create-page-bids-list-as-human)

File: src/pages/App.tsx [hunk 1]
```diff
@@ -59,6 +60,14 @@ function Main() {
...
             />
           </Route>
+          <Route path="/human/bids">
+            <Bids
+              accounts={accounts}
+              activeAccount={activeAccount}
+              setAccounts={setAccounts}
+              setActiveAccount={setActiveAccount}
+            />
+          </Route>
```
Comment (murilooon) [inline src/pages/App.tsx:70]: After merging https://github.com/KappaSigmaMu/ksm-app/pull/30 this Route will need to be updated

File: src/pages/human/Bids/Page.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  setAccounts: (accounts: { name: string | undefined; address: string }[]) => void
+}): JSX.Element => {
+  const { api } = useSubstrate()
+  const [bids, setBids] = useState<Vec<PalletSocietyBid> | []>([])
+
+  useEffect(() => {
+    if (!api) { return }
+    api.query.society.bids().then((response: Vec<PalletSocietyBid>) => {
+      setBids(response)
+    })
```
Comment (murilooon) [inline src/pages/human/Bids/Page.tsx:28]:
```suggestion
    if (api) {
      api.query.society.bids().then((response: Vec<PalletSocietyBid>) => {
        setBids(response)
      })
    }
```

WDYT?

File: src/components/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const Menu = styled.li`
+  list-style: none;
+`
+
+const MenuItem = ({ to, children }: { to: string; children: string }): JSX.Element => (
+  <li>
+    <NavLink
+      to={to}
+      activeStyle={{ color: 'white', fontSize: 20 }}
+      style={{ color: '#707070', fontSize: 16, lineHeight: 2, fontWeight: 700, textDecoration: 'none' }}
```
Comment (murilooon) [inline src/components/Sidebar.tsx:22]: I think we can use Styled Components here and get the color from global, WDYT?

File: src/components/BidsList.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+              style={{ fontWeight: 700, color: '#C4C4C4', backgroundColor: '#222222', borderColor: '#222222' }}
+            >
+              Lowest&nbsp;&nbsp;&nbsp;&nbsp;▾
+            </Button>
+            <Button className="me-2" variant="outline-dark" style={{ color: '#707070', borderColor: 'transparent' }}>
+              Newest&nbsp;&nbsp;&nbsp;&nbsp;▾
+            </Button>
+          </ButtonToolbar>
+        </Col>
+      </Row> */}
```
Comment (murilooon) [inline src/components/BidsList.tsx:42]: Something happen here?

File: src/components/BidsList.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    <Col xs={4} className="text-start">Wallet Hash</Col>
+    <Col xs={5} className="text-start">Bid Kind</Col>
+    <Col xs={2} className="text-end" style={{ paddingRight: 36 }}>Value</Col>
+  </Row>
+)
+
+const DataRow = ({ item }: { item: any }) => (
+  <Row
+    key={item.who?.toString()}
+    style={{ color: '#fff', lineHeight: 3, marginTop: 5, backgroundColor: '#343A40', borderRadius: 6 }}
```
Comment (murilooon) [inline src/components/BidsList.tsx:59]: Styled Components

File: src/components/BidsList.tsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+            </Button>
+          </ButtonToolbar>
+        </Col>
+      </Row> */}
+    </>
+  )
+}
+
+const Header = () => (
+  <Row style={{ color: '#fff', lineHeight: 3 }}>
```
Comment (murilooon) [inline src/components/BidsList.tsx:48]: Styled Components

File: src/components/BidsList.tsx [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    <Col xs={1} className="text-center">#</Col>
+    <Col xs={4} className="text-start">Wallet Hash</Col>
+    <Col xs={5} className="text-start">Bid Kind</Col>
+    <Col xs={2} className="text-end" style={{ paddingRight: 36 }}>Value</Col>
```
Comment (murilooon) [inline src/components/BidsList.tsx:52]: Styled Components

File: src/components/BidsList.tsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    <Col xs={1} className="text-center">
+      <Identicon value={item.who} size={32} theme={'polkadot'} />
+    </Col>
+    <Col xs={4} className="text-start text-truncate">
+      {truncateAccountId(item.who?.toString())}
+    </Col>
+    <Col xs={5} className="text-start text-truncate">
+      {humanizeBidKind(item.kind)}
+    </Col>
+    <Col xs={2} className="text-end" style={{ paddingRight: 36 }}>
```
Comment (murilooon) [inline src/components/BidsList.tsx:70]: Styled Components - could use the same styled component as other above

**Review by murilooon — CHANGES_REQUESTED** [review]
Pointed some nits. Also, this PR will need to be updated before merging.

**Review by murilooon — APPROVED** [review]

**Comment by alfakini** [comment]
[[PAGE] Implement the Bids Lists page](https://trello.com/c/JEUF7Iik/180-page-implement-the-bids-lists-page)

**Pull request merged**